api/src/models/index.js

```diff
@@ -1,5 +1,17 @@
 // Global Importer for models.
 
-exports.audit = require('./audit.js');
-exports.record = require('./record.js');
-exports.task = require('./task.js');
+// general
+exports.audit = require('./audit');
+exports.task = require('./task');
+
+// master
+exports.order = require('./master/order');
+exports.inspection = require('./master/inspection');
+
+// lng
+exports.orderLNG = require('./lng/order-lng');
```

Comment:
This should be importing the directory, by specifying an index.js, instead of the specific items in this index.js (same for next few lines)

Reply:
Fixed.